.github/workflows/lint.yaml

```diff
@@ -35,7 +35,7 @@ jobs:
         uses: golangci/golangci-lint-action@3a919529898de77ec3da873e3063ca4b10e7f5cc # v3.7.0
         with:
           version: v${{ needs.tools.outputs.golangci-lint-version }}
-          args: --out-format checkstyle:golangci-lint-report.xml
+          args: --out-format checkstyle:golangci-lint-report.xml --build-tags=other_docker_tests,besu_tests,geth_tests,erigon_tests,nethermind_tests,eth_env_tests
```

Comment:
Similar to other comments we've mentioned, I've had many a bad experience with build tags and tests, and would rather we only use them if absolutely necessary.